datasets/gov/vat/r22_pluost_kanap.csv

```diff
@@ -0,0 +1,16 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/vatzum/pluostines_kanapes,,,,,,,,,,,,,
+,,,,,,prefix,spinta,,,,,https://github.com/atviriduomenys/spinta/issues/,,
+,,,,,,,manifest,,,,,https://github.com/atviriduomenys/manifest/issues/,,
+,,,,,,,,,,,,,,
+,,,,Tiekejas,,,eiles_nr,,,4,open,,"Pateikiama informacija apie pluoštinių kanapių produktus tiekiančius ūkio subjektus (vardas, pavardė/ pavadinimas, juridinio asmens kodas, el. p., tel., tiekiamos produkcijos rūšys)",
+,,,,,eiles_nr,string,,,,4,open,,Eilės numeris,
+,,,,,tiekejas,string,,,,2,open,,"Tiekėjo vardas, pavardė / pavadinimas",
+,,,,,,comment,type,,"update(base: ""/datasets/gov/rc/jar/iregistruoti/JuridinisAsmuo"", type: ""ref"", ref: ""/datasets/gov/rc/jar/iregistruoti/JuridinisAsmuo"")	",4,open,"spinta:205, manifest:1290",,
+,,,,,tiekejo_nr,string,,,,2,open,,Tiekėjui suteiktas Nr.,
```

Comment:
Kodėl nurodote antrą brandos lygį?
Ar tai yra VAT sistemos suteikiamas nr?

Reply:
Tiekėjo_nr man atrodo nepateikiamas vieningu formatu: vieni numeriai turi vienokį šabloną (e.g. 17P-26PK), kiti - kitą (e.g. PKT-32). Dėl to priskyriau antrą brandos lygį. Ir taip, šis nr yra suteikiamas VAT.
![image](https://user-images.githubusercontent.com/73559010/229824988-2303bc61-6fc1-4d28-9425-0fb8c96b9a78.png)